tests/VeresOne.spec.js

```diff
@@ -70,6 +70,65 @@ describe('methods/veres-one', () => {
         .to.eql(JSON.stringify(UNREGISTERED_DOC, null, 2));
     });
 
+    it('should return a key present in an un-registered DID', async () => {
+      nock('https://ashburn.capybara.veres.one')
+        .get(`/ledger-agents`)
+        .reply(200, LEDGER_AGENTS_DOC);
+
+      const {ledgerAgent: [{service: {ledgerQueryService}}]} =
+        LEDGER_AGENTS_DOC;
+      nock(ledgerQueryService)
+        .post('/?id=' + encodeURIComponent(UNREGISTERED_NYM))
+        .reply(404);
+
+      _nockLedgerAgentStatus();
+
+      // eslint-disable-next-line max-len
+      const unregisteredKey = 'did:v1:test:nym:z6MkesAjEQrikUeuh6K496DDVm6d1DUzMMGQtFHuRFM1fkgt#z6MkesAjEQrikUeuh6K496DDVm6d1DUzMMGQtFHuRFM1fkgt';
+      const result = await v1.get({did: unregisteredKey});
+
+      expect(result.doc).to.eql({
+        '@context': [
```

Comment:
Is it correct / acceptable, to use the DID contexts with a returned key object?

Reply:
Well, let's say yes *for now*. That might need to change in the future based on how the contexts get scoped when they are updated to JSON-LD 1.1. But being able to include the contexts with the keys is an important consideration for that process when we get there. I think it's fine for now.